tests/ExamplePluginWithLib/Upstream/Library/CMakeLists.txt

```diff
@@ -70,7 +64,7 @@ target_link_libraries(
 )
 
 # make an alias to have acces to the target in a shared project build-tree
-add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
+#add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```

Comment:
Why is this line commented ? How is the namespace generate ?

Reply:
This line is no more needed (and I delete it now). The function `configure_radium_library` used to configura any Radium library does this, line 684 of `radium_setup_functions.cmake`.